File: app/src/main/java/com/nicholasdoglio/eyebleach/ui/photodetail/PhotoDetailFragment.java
package com.nicholasdoglio.eyebleach.ui.photodetail;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.nicholasdoglio.eyebleach.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class PhotoDetailFragment extends Fragment {


    public PhotoDetailFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_photo_detail, container, false);
    }

}

File: mobile/src/main/java/com/nicholasdoglio/eyebleach/ui/photodetail/PhotoDetailFragment.kt
package com.nicholasdoglio.eyebleach.ui.photodetail


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.nicholasdoglio.eyebleach.R
import com.nicholasdoglio.eyebleach.util.inflate


class PhotoDetailFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? = container?.inflate(R.layout.fragment_photo_detail)
}